courses/intro-to-spark/book/01_Overview.rst

```diff
@@ -254,8 +254,9 @@ while ``Incr_And_Log`` is not as it attempts to update the global variable
 No aliasing of names
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Another restriction imposed in the SPARK subset concerns aliasing. There are
-two reasons to forbid aliasing in SPARK:
+Another restriction imposed in the SPARK subset concerns aliasing. We say that
+two names are aliased if they refer to the same object. There are two reasons
```

Comment:
Maybe we could also link to a Wikipedia page? https://en.wikipedia.org/wiki/Aliasing_(computing)
Just an idea, feel free to drop it.